### DISCUSSION TITLE
popupMatchSelectWidth 能否支持 rem 单位 ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42694] [Q&A]
目前 只支持 true 和  number,无法使用 rem 单位
<img width="494" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/0f4865fe-e91d-429f-8760-358808ab5bef">

我使用的 mantine + antd,   mantine 使用的是rem 单位
是不是 antd  全局使用 px? 那这样的话, 是不是我也没必要使用 rem 吗?

**Comment by RedJue** [comment]
rem 可以用其他第三方 px2rem 库转掉就行。

**Reply by vaynevayne** [reply]
antd  内部的 也能转?

**Reply by yoyo837** [reply]
编译的时候让他把antd一起编译就可以了，但不确定是否影响antd正常表现

**Reply by vaynevayne** [reply]
我得自己写编译逻辑,自己控制antd  的输出文件位置等等吗?

**Reply by RedJue** [reply]
不需要吧 第三方库应该可以指定的 不需要自己写逻辑

**Comment by vaynevayne** [comment]
话说这个属性就是赋值给width属性吗？ 能不能使用cal (100% )? 这里的100% 能指代输入框的宽度吗
